modules/vector-index/pages/use-vector-indexes.adoc

```diff
@@ -0,0 +1,273 @@
+= Use Vector Indexes for AI Applications
```

Comment:
See comment on [vectors-and-indexes-overview.adoc][1]. Should that page actually be titled "Use Vector Indexes for AI Applications", and this file be titled "Vectors and Vector Indexes Overview"?

(Again, I'm not suggesting you rename the files, as that leads to a risk of broken links. Or that you should swap the files in the nav, because this is clearly the right content for the second page in this section. Just the doc titles.)

[1]: https://github.com/couchbaselabs/docs-devex/pull/325/files#diff-675176b8a2bf2dd587e6e6752c443d27766c8909f01cc0c2bdd1db1054572f9d